### DISCUSSION TITLE
How to enable fullscreen?

### DISCUSSION THREAD

**Discussion by ConvergeMedia** [LunatiqueCoder/react-native-media-console#69] [Q&A]
I was working on this and everything else works exept the fullscreen button

**Comment by LunatiqueCoder** [comment]
You have to implement the fullscreen functionality by yourself. We don’t provide it because it can be different from project to project

**Comment by vegerot12** [comment]
How to implement this, can you please tell which is the prop or callback that fires when I press the fullScreen button.
